cmd/apps/nginx_app.go

```diff
@@ -28,6 +28,7 @@ flag and the ingress-nginx docs for more info`,
 	nginx.Flags().StringP("namespace", "n", "default", "The namespace used for installation")
 	nginx.Flags().Bool("update-repo", true, "Update the helm repo")
 	nginx.Flags().Bool("host-mode", false, "If we should install ingress-nginx in host mode.")
+	nginx.Flags().Bool("default-ingress", true, "Is this the default ingressClass for the cluster?")
```

Comment:
If we set this as default isn't this a backward incompatible change?

I think it would be better with default `false`.

Reply:
but what's the point for it being false? i thought that idea is to preconfigure application to be up&fully functional for development/testing purposes.

Reply:
To be clear, your approach makes more sense, if we have no promise for backward compatibility (which afaik we doesn't) you are right, it is better this way then.